internal/optimizer/normalizecharactersoptimizer_test.go

```diff
@@ -0,0 +1,327 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package optimize
+
+import (
+	"github.com/konveyor/move2kube/internal/types"
+	plantypes "github.com/konveyor/move2kube/types/plan"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"reflect"
+	"testing"
+)
+
+func TestStripQuotation(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+
+	t.Run("strip matching single quotation marks from input string", func(t *testing.T) {
+		// Setup
+		inputString := "'testString'"
+		want := "testString"
+
+		// Test
+		actual := stripQuotation(inputString)
+		if !reflect.DeepEqual(actual, want) {
+			t.Fatal("Failed to get the expected properly. Expected:", want, "actual:", actual)
```

Comment:
@HarikrishnanBalagopal Can we use  https://pkg.go.dev/github.com/google/go-cmp/cmp#Diff instead of just printing expected and actual here as well? 
In order to maintain consistency throughout codebase and sometimes dealing with strings to find difference is also hectic.

Reply:
yes